client/my-sites/email/email-providers-comparison/price/google-workspace.tsx

```diff
@@ -62,7 +62,8 @@ const GoogleWorkspacePrice = ( {
 	}
 
 	const isDiscounted = hasDiscount( product );
-	const isEligibleForFreeTrial = isDomainEligibleForGoogleWorkspaceFreeTrial( domain );
+	const isEligibleForFreeTrial =
+		isDomainInCart || isDomainEligibleForGoogleWorkspaceFreeTrial( domain );
```

Comment:
I don't think we can do that, here is what I get when I try to purchase Google Workspace with a new domain when the server patch is not applied (so targeting the old billing plans):

![image](https://user-images.githubusercontent.com/594356/182800210-9b9da6da-f2f0-47c4-8c68-3fd3f4679a50.png)

With Titan, we assumed that we would always offer a free trial, which is not the case with Google Workspace.

Reply:
Good catch!